go/vt/vtenv/vtenv.go

```diff
@@ -0,0 +1,75 @@
+package vtenv
+
+import (
+	"vitess.io/vitess/go/mysql/collations"
+	"vitess.io/vitess/go/mysql/config"
+	"vitess.io/vitess/go/vt/sqlparser"
+)
+
+type Environment struct {
+	collationEnv   *collations.Environment
+	parser         *sqlparser.Parser
+	mysqlVersion   string
+	truncateUILen  int
+	truncateErrLen int
```

Comment:
These two feel a bit out of place, because they seem very specific to the status pages. 🤷🏻‍♂️

Reply:
They are used in a few more places but yeah. I don’t have a much better idea though, we need to pass them all the through then.